src/playback/summary.jsx

```diff
@@ -200,7 +200,7 @@ class PlaybackSummary extends React.Component {
     const {
       bits, encoding, classes, browserId,
     } = this.props;
-    const platformType = browserId === 'firefox' ? 'desktop' : 'mobile';
+    const platformType = browserId === 'firefox' || browserId === 'chrome' ? 'desktop' : 'mobile';
```

Comment:
This logic is a little messy.  We should have this desktop/mobile distinction in our metadata somewhere.

No need to change it, but keep it in mind for the future